examples/retrieve_customer.go

```diff
@@ -1,20 +1,33 @@
 package main
 
 import (
+	"fmt"
+	"os"
+	"encoding/json"
 	"github.com/ncr4/tuneuptechnology-go"
 )
 
 func main() {
-	tuneuptechnology.RetrieveCustomer(
-		// Pass in your email and API key
-		&tuneuptechnology.Client{
-			Auth: "",
-			APIKey: "",
-		},
+	// Setup your email and API key
+	api_email := "EMAIL_HERE"
+	api_key := "API_KEY_HERE"
 
-		// retrieves customer with ID #23
+	// Retrieve a single customer
+	customer := tuneuptechnology.RetrieveCustomer(
 		&tuneuptechnology.Customer{
-			Id: 23,
+			Auth: api_email,
+			APIKey: api_key,
+			Id: 1, // the ID of the customer you are retrieving
 		},
 	)
+
+	// Print the response as pretty JSON
```

Comment:
Lots of duplication in the examples, consider a `utils` module that formats the responses into JSON to remove the extra code from examples, leaving only the showcase of the current lib.

Reply:
Added a separate function to pretty print, fantastic idea!